service/src/main/resources/db/changesets/20230410_schema_reset.yaml

```diff
@@ -441,25 +473,25 @@ databaseChangeLog:
                   remarks: Deleting a concept set will cascade to delete the criteria contained in it
               - column:
                   name: display_name
-                  type: text
+                  type: ${text.type}
               - column:
                   name: plugin_name
-                  type: text
+                  type: ${text.type}
                   constraints:
                     nullable: false
               - column:
                   name: selection_data
-                  type: text
+                  type: ${text.type}
                   constraints:
                     nullable: false
               - column:
                   name: ui_config
-                  type: text
+                  type: ${text.type}
```

Comment:
These two fields are serialized json and may be longer than 255 characters. What do you think about using `longtext` for these fields in MySql?

Reply:
Several alternatives: varchar(65xxx), longtext can hold up to 4GB data

Reply:
OK either sounds fine: longer varchar (maybe 25k is enough) or longtext. We can always change the datatype in the future if we find we're hitting the limit. I think we'd hit the 255 limit fairly quickly though.